--stress

```diff
@@ -0,0 +1,875 @@
+PASSED: //pkg/ccl/jobsccl/jobsprotectedtsccl:jobsccl_test (see /private/var/tmp/_bazel_adityamaru/2592a7600a28e43d0b132c217219adf9/execroot/cockroach/bazel-out/darwin-fastbuild/testlogs/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsccl_test/test.log)
```

Comment:
file included accidentally?

Reply:
doh, indeed. Going to wait for CI to see if it shakes anything out before pushing.